.travis.yml

```diff
@@ -1,37 +1,60 @@
 language: java
-sudo: false
-jdk:
-- openjdk8
-- openjdk9
-- openjdk10
-- openjdk11
+
 cache:
   directories:
-  - ".autoconf"
-  - "$HOME/.m2"
+    - ".autoconf"
+    - "$HOME/.m2"
+
+before_install:
+  - wget https://github.com/sormuras/bach/raw/master/install-jdk.sh
+
+matrix:
+  include:
+    # OpenJDK
+    - env:
+        - JDK='OpenJDK 8'
+      os: linux
+      dist: xenial
```

Comment:
does it really makes sense to matrix test `dist`?

Reply:
removed